engine/api/project.go

```diff
@@ -4,20 +4,21 @@ import (
 	"database/sql"
 	"net/http"
 	"regexp"
+	"strings"
 	"time"
 
 	"github.com/go-gorp/gorp"
 	"github.com/gorilla/mux"
 
 	"github.com/ovh/cds/engine/api/businesscontext"
 	"github.com/ovh/cds/engine/api/group"
+	"github.com/ovh/cds/engine/api/permission"
 	"github.com/ovh/cds/engine/api/project"
 	"github.com/ovh/cds/sdk"
 	"github.com/ovh/cds/sdk/log"
 )
 
 func getProjectsHandler(w http.ResponseWriter, r *http.Request, db *gorp.DbMap, c *businesscontext.Ctx) error {
-	// Get project name in URL
 	withApplication := FormBool(r, "application")
```

Comment:
why deleting comments ?

Reply:
Because it's false :) We don't get project name in URL on this handler

Reply:
indeed
or fix it :)

Reply:
other comments are back